shell/platform/android/io/flutter/plugin/platform/PlatformViewsController.java

```diff
@@ -47,11 +43,51 @@
     // The texture registry maintaining the textures into which the embedded views will be rendered.
     private TextureRegistry mTextureRegistry;
 
-    // The messenger used to communicate with the framework over the platform views channel.
-    private BinaryMessenger mMessenger;
+    // The system channel used to communicate with Flutter about platform views.
```

Comment:
I'd keep it saying: to communicate with "the framework", this code is also part of Flutter.

Reply:
done